documents/meetup-starter-kit.md

```diff
@@ -0,0 +1,82 @@
+## Structure/Format
+
+Two to three presenters has been found to be best by multiple meetup coordinators.
+* 20-30 minute talks for each presenter is the sweet spot.
+  * Presentations run the risk of becoming too lengthy, and may lose interest of the audience.
+* You can specifically structure your event to a set number of speakers
+  * two speakers might be beneficial to give both a basic and an advanced talk, catering to the
+two groups.
+* Over your first few events, try different formats (two 30 minutes, three 20 minutes, two 20 minutes and 20 minutes of Q&A, and so on) and see which ones you think work best.
+
+As you start, solicit developers to come present at your meetup. You should
+plan out speakers a few events out. You want to be sure that you have people
```

Comment:
Which line? 11? I'm not seeing line numbers for your comments, just a block of 11 lines. If you could clarify which line you meant that would be very helpful.